enforcer/netfilter/netfilter_linux.go

```diff
@@ -44,26 +44,26 @@ const (
 	AfInet = 2
 
 	//NfDrop Net filter verdict
-	NfDrop verdictType = 0
```

Comment:
We need to control the values. Defined in the kernel. iota is not deterministic.